.pipelines/templates/template-podman.yml

```diff
@@ -0,0 +1,167 @@
+parameters:
+  - name: jobName
+    type: string
+  - name: makeCommand
+    type: string
+  - name: repository
+    type: string
+  - name: acrFQDN
+    type: string
+  - name: imageRepoName
+    type: string
+  - name: pushLatest
+    type: boolean
+    default: false
+
+jobs:
+  - job: ${{ parameters.jobName }}
+    pool:
+      name: 1es-aro-ci-pool
+    variables:
+      ACR_FQDN: ${{ parameters.acrFQDN }}
+      REPOSITORY: ${{ parameters.repository }}
+      IMAGE_TAG: $(Build.BuildId)  # Include the unique BuildId or another identifier
+      PODMAN_REMOTE_ARGS: '--remote --url=tcp://localhost:8888'  # Define the standard Podman remote arguments
+
+    steps:
+      - template: ../templates/template-checkout.yml
+
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: 'ado-pipeline-dev-image-push'  # Service connection name
+          scriptType: 'bash'
+          scriptLocation: 'inlineScript'
+          inlineScript: |
+            echo "Retrieving ACR token..."
+            REGISTRY_NAME=${ACR_FQDN%%.*}
+
+            if [ -z "$REGISTRY_NAME" ]; then
+              echo "Error: ACR_FQDN is not set correctly."
+              exit 1
+            fi
+            TOKEN=$(az acr login --name $REGISTRY_NAME --expose-token --output tsv --query accessToken)
+
+            if [ -z "$TOKEN" ]; then
+              echo "Error: Failed to retrieve ACR token."
+              exit 1
+            fi
+
+            echo "##vso[task.setvariable variable=ACR_TOKEN]$TOKEN"
+
+      - script: |
+          set -xe
+          export NO_CACHE=true
+          export TOKEN=$(ACR_TOKEN)
+          export REPOSITORY=${{ parameters.repository }}
+          export IMAGE_REPO_NAME=${{ parameters.imageRepoName }}
+
+          # Install dependencies (podman, azure-cli, go, libicu, gocover-cobertura)
+          echo "Installing required tools..."
+          sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+          sudo dnf config-manager --add-repo https://packages.microsoft.com/config/rhel/8/prod.repo
+          sudo dnf install -y azure-cli podman make libicu
```

Comment:
q: interesting - do we need `libicu`? That package doesn't sound familiar to me - just curious what it is?

Reply:
removed the libicu